web/client/src/library/components/editor/Editor.tsx

```diff
@@ -95,7 +95,7 @@ function EditorMain({ tab }: { tab: EditorTab }): JSX.Element {
     return [
       ...defaultKeymapsEditorTab,
       {
-        key: 'Ctrl-Enter',
+        key: 'Shift-Mod-Enter',
```

Comment:
What's Mod?

Reply:
Codemirror using `Mod` a shorthand for Cmd- on Mac and Ctrl- on other platforms. So Mod-b is Ctrl-b on Linux but Cmd-b on macOS.